### ISSUE TITLE
libglog.pc.in should include libunwind as private dep for static linking case

### ISSUE THREAD

**Issue by Katerall** [google/glog#57]
This was reported along with a patch by someone before (+CLA):
https://code.google.com/p/google-glog/issues/detail?id=207

I ran into this issue during compilation with errors similar to those reported in the issue:

```
/local/adp/unwind-compile-error/protobuf-util/vendor/src/glog-build/../glog/src/stacktrace_libunwind-inl.h:65: undefined reference to `_Ux86_64_getcontext'
/local/adp/unwind-compile-error/protobuf-util/vendor/src/glog-build/../glog/src/stacktrace_libunwind-inl.h:66: undefined reference to `_ULx86_64_init_local'
/local/adp/unwind-compile-error/protobuf-util/vendor/src/glog-build/../glog/src/stacktrace_libunwind-inl.h:78: undefined reference to `_ULx86_64_step'
/local/adp/unwind-compile-error/protobuf-util/vendor/src/glog-build/../glog/src/stacktrace_libunwind-inl.h:70: undefined reference to `_ULx86_64_get_reg'
```

Applying the patch worked for me. Please fix.

**Comment by amosbird** [comment]
The patch gets lost in the original url. I don't see related fixes, however, it's mentioned as fixed in 0.5. Any references?

**Comment by sergiud** [comment]
The issue was closed without a fix. Please submit a PR.
